src/util/mpir_strerror.c

```diff
@@ -14,24 +14,13 @@ int strerror_r(int errnum, char *strerrbuf, size_t buflen);
 #endif
 #endif
 
-/* provides a thread-safe version of strerror */
-const char *MPIR_Strerror(int errnum)
+/* wrapper for strerror_r to return the buffer used */
+const char *MPIR_Strerror(int errnum, char *strerrbuf)
```

Comment:
I am a bit concerned that we are assuming the buffer size by convention here. How about we further add the parameter of buffer size?

Reply:
Yes, good point.  Such assumptions are bad coding practice.  I'll fix it.